packages/moxb/package.json

```diff
@@ -49,12 +49,14 @@
     "tslib": "^1.9.0"
   },
   "devDependencies": {
+    "@types/atob": "2.1.2",
```

Comment:
`@types/atob` is inluded here, but in the dependencies `btoa-lite` is imported. Are they using the same ts type definitions?